src/database/adatabasetypes.h

```diff
@@ -19,38 +19,41 @@
 #define DECLARATIONS_H
 
 #include <QtCore>
-#include "src/oplconstants.h"
 #include "src/testing/adebug.h"
 
-/// \todo Break apart these aliases to their
-/// corresponding sub section of the program
-/// eg DB related to ADatabase.h and so on
+// [G]: TODO. Break apart these aliases to their
+// corresponding sub section of the program
+// eg DB related to ADatabase.h and so on
 
-/*!
- * \brief An alias for QString
- *
- * Very long description *with* **markdown?**
- * - - -
- * # Header
- */
-using ColName = QString;
-using ColData = QVariant;
-using TableName = QString;
-using RowId = int;
+using RowId_t = int;
+using PilotName_t = QString;
+using PilotRowId_t = RowId_t;
+using TailRgstr_t = QString;
+using TailId_t = RowId_t;
+using ArprtICAO_t = QString;
+using ArprtIATA_t = QString;
+using ArprtName_t = QString;
+using ArprtId_t = RowId_t;
+using ColName_t = QString;
+using ColData_t = QVariant;
+using TableName_t = QString;
+using RowId_t = int;
```

Comment:
Let's spell out the words here

`TailRegistration`,`AirportIATA` etc.

Reply:
I find it funny that a pilot whose entire life is abbreviations wants me to un-abbreviate common fraces like registration and airport

Reply:
I'm not against abbreviations, they are useful! But neither `Arprt` nor `Rgstr` are (common) abbreviations. Those are like writing `QVrnt` instead of `QVariant`.

Abbreviating and omission of vowels are two different things.

Abbreviation is fine, it would be insane to write `AirportInternationalAirTransportAssociation` instead of `AirportIATA`. That's because IATA or ICAO etc. aren't regular words.

But disemvoweling is different (and bad). After all, we're not writing a SMS limited to 160 characters.